test/hotspot/jtreg/compiler/codegen/TestRedundantLea.java

```diff
@@ -331,48 +333,65 @@ public void testPhiSpill() {
         this.classArr8bit[OFFSET8BIT_IDX] = new StoreNTestHelper(CURRENT, OTHER);
         this.classArr8bit[OFFSET32BIT_IDX] = new StoreNTestHelper(CURRENT, OTHER);
     }
+}
+
+class StoreNTestHelper {
+    Object o1;
+    Object o2;
+
+    public StoreNTestHelper(Object o1, Object o2) {
+        this.o1 = o1;
+        this.o2 = o2;
+    }
+}
+
+// This test validates that the peephole removes simple spills.
+// The code for the test originates from compiler/escapeAnalysis/Test6775880.java.
+class SpillTest {
+    int cnt;
+    int b[];
+    String s;
+
+    @Run(test = "test")
+    public static void run() {
+        SpillTest t = new SpillTest();
+        t.cnt = 3;
+        t.b = new int[3];
+        t.b[0] = 0;
+        t.b[1] = 1;
+        t.b[2] = 2;
+        int j = 0;
+        t.s = "";
+        t.test();
+    }
 
     @Test
+    // TODO: Make tests more precise
```

Comment:
I guess that was already there for mainline but I'm wondering, is there a tracking issue for that?